oci/airflow-rocks/documentation.yaml

```diff
@@ -0,0 +1,144 @@
+version: 1
+
+application: apache airflow
+description: >
+  Apache Airflow is a platform to programmatically author, schedule, and monitor workflows. This rock ("airflow-rock") packages Airflow built from source on Ubuntu with Python and a wide set of providers.
+
+docker:
+  parameters:
+    - -p 8080:8080
+    - -e AIRFLOW_HOME=/var/lib/airflow
+    - -v /path/to/dags:/var/lib/airflow/dags
+    - -v /path/to/airflow.cfg:/var/lib/airflow/airflow.cfg
```

Comment:
Should we rather go with `/opt/airflow/`? That's also the configuration they have in upstream Airflow (see [here](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#fetching-docker-compose-yaml)).